homeassistant/components/smarttub/binary_sensor.py

```diff
@@ -98,3 +114,9 @@ def extra_state_attributes(self):
     def device_class(self) -> str:
         """Return the device class for this entity."""
         return DEVICE_CLASS_PROBLEM
+
+    async def async_snooze(self, **kwargs):
```

Comment:
We don't need to accept kwargs here. We can just name the parameter the same as the service data attribute, ie `days`.

Reply:
https://github.com/home-assistant/core/pull/51093